scripts/irods/test/test_chunkydevtest.py

```diff
@@ -112,8 +112,8 @@ def test_beginning_from_devtest(self):
         self.admin.assert_icommand("imeta add -d " + irodshome + "/icmdtest/foo1 testmeta2 hello")
         self.admin.assert_icommand("imeta ls -d " + irodshome + "/icmdtest/foo1", 'STDOUT_SINGLELINE', ["testmeta1"])
         self.admin.assert_icommand("imeta ls -d " + irodshome + "/icmdtest/foo1", 'STDOUT_SINGLELINE', ["hello"])
-        self.admin.assert_icommand("imeta qu -d testmeta1 = 180", 'STDOUT_SINGLELINE', "foo1")
-        self.admin.assert_icommand("imeta qu -d testmeta2 = hello", 'STDOUT_SINGLELINE', "dataObj: foo1")
+        self.admin.assert_icommand("iquest 'collection: %s\ndataObj: %s' \"SELECT COLL_NAME, DATA_NAME WHERE META_DATA_ATTR_VALUE = '180' AND META_DATA_ATTR_NAME = 'testmeta1'\"", 'STDOUT_SINGLELINE', "foo1")
+        self.admin.assert_icommand("iquest 'collection: %s\ndataObj: %s' \"SELECT COLL_NAME, DATA_NAME WHERE META_DATA_ATTR_VALUE = 'hello' AND META_DATA_ATTR_NAME = 'testmeta2'\"", 'STDOUT_SINGLELINE', "dataObj: foo1")
```

Comment:
looks weird in this order... can we flip em?

```suggestion
        self.admin.assert_icommand("iquest 'collection: %s\ndataObj: %s' \"SELECT COLL_NAME, DATA_NAME WHERE META_DATA_ATTR_NAME = 'testmeta1' AND META_DATA_ATTR_VALUE = '180'\"", 'STDOUT_SINGLELINE', "foo1")
        self.admin.assert_icommand("iquest 'collection: %s\ndataObj: %s' \"SELECT COLL_NAME, DATA_NAME WHERE META_DATA_ATTR_NAME = 'testmeta2' AND META_DATA_ATTR_VALUE = 'hello'\"", 'STDOUT_SINGLELINE', "dataObj: foo1")
```